src/EPR.Calculator.Frontend/Constants/StaticHelpers.cs

```diff
@@ -10,5 +10,6 @@ public class StaticHelpers
         public const string MimeType = "test/csv";
         public const long MaxFileSize = 50 * 1024; // 50 KB
         public const string MediaType = "application/json";
+        public const string CalculationRunNameEmpty = "Enter a name for this calculation";
```

Comment:
Please remove this constant from here and create a new constant file and call it ErrorMessages.cs

Reply:
@udaydefra We already using this static helper class for other validations so I thought I can use this file to add my validation. If you want I can move all validations to separate file for validations

Reply:
yes please do

Reply:
**done**